examples/pylint/E0213_no_self_argument.py

```diff
@@ -0,0 +1,9 @@
+class MyClass:
```

Comment:
Question: how is this different from the previous error? Will this error be emitted if the first argument is something other than `self`?